sympy/algebras/quaternion.py

```diff
@@ -507,6 +546,10 @@ def rotate_point(pin, r):
     def to_axis_angle(self):
         """Returns the axis and angle of rotation of a quaternion
 
+        Returns
+        =======
+        tuple
```

Comment:
It may contain some description if possible
<img src="https://user-images.githubusercontent.com/34944973/52392064-a1159600-2ae3-11e9-81f1-f1925afa5787.png" width="500px">